controllers/keyword.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"net/http"
+
+	. "google-scraper/helpers"
+	. "google-scraper/models"
+	"google-scraper/presenters"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+)
+
+type KeywordController struct {
+	baseController
+}
+
+func (c *KeywordController) Show() {
+	web.ReadFromRequest(&c.Controller)
+	keyword, err := c.findKeyword()
+	if err != nil {
+		c.Ctx.Redirect(http.StatusFound, "/")
+		return
+	}
+
+	result := keyword.IsBelongTo(c.CurrentUser)
+	if !result {
+		c.Ctx.Redirect(http.StatusFound, "/")
```

Comment:
Shouldn't there be a least a flash message to inform the user that something happened? Another alternative is to return unauthorized error.

Reply:
Added flash message support